src/js/actions/ui.js

```diff
@@ -570,12 +600,18 @@ define(function (require, exports) {
                 return this.flux.actions.ui.zoomInOut({ "zoomIn": true, "preserveFocus": true });
             }.bind(this));
 
-        return Promise.join(osPromise, owlPromise, pathPromise, zoomInShortcutPromise)
+        // Initialize the reference point from preferences
+        var preferences = this.flux.store("preferences"),
+            referencePoint = preferences.get(REFERENCE_POINT_PREFS_KEY, "lt"),
```

Comment:
mystic string. maybe consider using a const instead. for example `DEFAULT_REFERENCE_POINT`